aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -857,6 +857,21 @@ Tensor _nested_view_from_buffer(
     storage_offsets);
 }
 
+Tensor _nested_from_buffer_unchecked(
```

Comment:
I think more description here around why we need a second one and how it's I eye red to be used would be helpful. At least for future me lol

Reply:
Yep, you got it! incoming